apps/web/pages/checkout.vue

```diff
@@ -120,8 +124,7 @@ const handlePaymentMethodUpdate = async (paymentMethodId: number) => {
 const order = async () => {
   const data = await createOrder({
     paymentId: paymentMethodData.value.selected,
-    // eslint-disable-next-line unicorn/expiring-todo-comments
-    shippingPrivacyHintAccepted: true, // TODO
+    shippingPrivacyHintAccepted: shippingPrivacyAgreement.value,
```

Comment:
We talked about it but leaving this here in case it helps anybody else:

https://vuejs.org/api/reactivity-utilities.html#torefs

`toRefs is useful when returning a reactive object from a composable function so that the consuming component can destructure/spread the returned object without losing reactivity`

For this reason our state properties from composables are reactive in our vue files.

Reply:
Maybe add this to the Wiki as well, so that it's easier to find again?